<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37,37,37);


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);


</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(232, 104, 30);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>pi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(232, 104, 30);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_30">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(232, 104, 30);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_31">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>460</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>       x = </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(232, 104, 30);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>611</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {


background-color: rgb(20, 20, 20);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QWidget:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>460</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(232, 104, 30);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(232, 104, 30);

size: 150px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(255, 255, 255);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
            background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;

}


QPushButton:hover {
             background-color: rgb(90, 89, 91);
}</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(138, 137, 140);


</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(138, 137, 140);


</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(138, 137, 140);


</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Сумма(руб.)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Срок (Мес)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Процент(%)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>110</y>
        <width>671</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>651</width>
          <height>371</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;

background-color: rgb(37, 37, 37);

color: rgb(138, 137, 140);

font-size: 33px;

font: 16pt &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
